modules/apps/client-extension/client-extension-type-api/src/main/java/com/liferay/client/extension/type/manager/CETManager.java

```diff
@@ -31,8 +31,16 @@ public CET addCET(
 	public CET getCET(long companyId, String externalReferenceCode);
 
 	public List<CET> getCETs(
-			long companyId, String keywords, String type, Pagination pagination,
-			Sort sort)
+			long companyId, boolean excludeInstanceScopedCETs, String keywords,
+			String type, Pagination pagination, Sort sort)
+		throws PortalException;
```

Comment:
If we change an existing method signature, all providers and consumers of this interface are forced to change. 

An alternative is to keep the existing method untouched and add a new one:
``` 
public List<CET> getCETs(
			long companyId, boolean excludeInstanceScopedCETs, String keywords,
			String type, Pagination pagination, Sort sort)
```
This way, we can turn the implementation of the old one as a special case of invocation of the new one. We can even do it in the interface itself as a default method, so that it'll be available for the implementations:

``` 
public defaul List<CET> getCETs(
			long companyId, String keywords, String type, Pagination pagination,
			Sort sort) {
          return getCETs(companyId, false, keywords, type, pagination, sort);
}
```
Now, `CETManagerImpl` would just implement the new method as you do [here](https://github.com/liferay-frontend/liferay-portal/pull/4416/files#diff-438b6f11f78bd05fac56fb7890e5e939bfe147f1c09d74b4784cf3ee6fdcdb56R104), and there is no need for it to implement the old one.

It's also possible to implement both methods in the implementation class. The point is that most usages in the consumers don't need this flag so both methods shall be made available. This way, we are not breaking potential client extension types implemented out there.